addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23751,11 +23751,7 @@ msgctxt "#40001"
 msgid "Manage versions"
 msgstr ""
 
-#. Manage video version dialog title
-#: xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp
-msgctxt "#40002"
-msgid "Convert into an additional version of {0:s}"
-msgstr ""
+#obsolete msgid 40002, replaced by msgid 40026
```

Comment:
We usually do not document this kind of change, because we have GitHub. ;-)
And I think translators cannot see these comments in Weblate anyway.